src/include/btmem.h

```diff
@@ -720,8 +720,9 @@ struct __wt_page {
 #define WT_PAGE_EVICT_LRU 0x010u          /* Page is on the LRU queue */
 #define WT_PAGE_EVICT_NO_PROGRESS 0x020u  /* Eviction doesn't count as progress */
 #define WT_PAGE_INTL_OVERFLOW_KEYS 0x040u /* Internal page has overflow keys (historic only) */
-#define WT_PAGE_SPLIT_INSERT 0x080u       /* A leaf page was split for append */
-#define WT_PAGE_UPDATE_IGNORE 0x100u      /* Ignore updates on page discard */
+#define WT_PAGE_IN_URGENT_QUEUE 0x080u    /* Page is in the urgent queue */
```

Comment:
If you decide to rename this flag, then you can have it next to the existing one `WT_PAGE_EVICT_LRU`.

Reply:
Updated in [de67431](https://github.com/wiredtiger/wiredtiger/pull/8950/commits/de67431b743a495ffb18d3a0d8900ee33f04a66e)